variables.tf

```
@@ -34,6 +50,18 @@ variable "provisionssmsessionmanager_policy_name" {
   type        = string
 }
 
+variable "read_lambda_bucket_policy_description" {
+  default     = "Allows read-only access read-only access to the bucket in the Terraform account containing Lambda deployments."
+  description = "The description to associate with the IAM role that allows read-only access read-only access to the bucket in the Terraform account containing Lambda deployments."
```

Remove duplicated words:
```suggestion
  default     = "Allows read-only access to the bucket in the Terraform account containing Lambda deployments."
  description = "The description to associate with the IAM role that allows read-only access to the bucket in the Terraform account containing Lambda deployments."
```

---

Please see commit [22984eb](https://github.com/cisagov/cool-accounts-userservices/pull/39/commits/22984ebcb00b4bebaf7af0200496d4012e163e56).